check/test_test.go

```diff
@@ -1,121 +0,0 @@
-// Copyright © 2017 Aqua Security Software Ltd. <info@aquasec.com>
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-package check
-
-import (
-	"io/ioutil"
-	"os"
-	"strings"
-	"testing"
-)
-
-var (
-	in       []byte
-	controls *Controls
-)
-
-func init() {
```

Comment:
Also let's make sure we keep these tests, they are useful

Reply:
See previous comments.

Reply:
I see we have something in bench_common called TestTestExecute, but I think we are losing the set of actual test cases below, for specific kube-bench tests. Don't we need those?